cmd/file.d.go

```diff
@@ -70,7 +72,7 @@ func main() {
 
 	_, _ = maxprocs.Set(maxprocs.Logger(logger.Debugf))
 
-	go listenSignals()
+	go listenSignals(context.Background())
```

Comment:
providing this function context.Background() is redundant, it can acquire it by itself. it has only meaning if custom context is provided.

Reply:
Done.